e2e/common_test.go

```diff
@@ -11,14 +11,16 @@ import (
 	"os/exec"
 	"strings"
 
-	"github.com/golang/glog"
 	. "github.com/onsi/gomega"
 	kubevirtv1 "kubevirt.io/api/core/v1"
+	kubecvirtcli "kubevirt.io/csi-driver/pkg/kubevirt"
```

Comment:
I see that this package is newly introduced and appears to be a copy of the kubevirt client-go pkg into the csi-driver repo.

I get why this was done. It's easier to copy this package into our repo here rather than fixing the kubevirt client-go dependencies. However, I'm concerned about maintaining this package in two places.

As alternative both vendoring kubevirt client-go and copying this port forwarding logic into our repo, i think we should use the virtctl command directly within our test driver to perform the port forwarding. The e2e tests already dependent on a few cli tools (kubecli, clustercli) as seen in `hack/run-e2e.sh`.  We could add virtctl there as well.

The result would be that the `e2e/common_test.go`'s `tenantClusterAccess` would be managing a virtctl process rather than a listener.